src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -29,9 +32,11 @@ private enum Size
         private readonly Encoding _encoding = Encoding.UTF8;
         internal readonly IOptionService _persistentEnabledOptionService = new OptionServiceMock(new Dictionary<IOption, object>
         {
-            { PersistentStorageOptions.Enabled, true }
+            { PersistentStorageOptions.Enabled, true },
+            { StorageOptions.SolutionSizeThreshold, 100 }
```

Comment:
100 whats?!?!?!  what's the unit?!?!

Reply:
I think "solution size units", which are currently implemented as bytes of source code? https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Desktop/Workspace/SolutionSize/SolutionSizeTracker.cs

(@heejaechang might want to chime in here.)

Reply:
yes. it is sum of length of sourceText. so not exactly bytes since for unicode, 2 bytes is 1 length. if source text's encoding is ascii code, it will be 1 byte for 1 length. 

this is not intended to be precise. rough estimate is all we need. we first tried to read file size to make it bytes, which show up at RPS file read, moved to tree length which must already there when solution crawler is called.